gapis/trace/tracer/tracer.go

```diff
@@ -87,7 +87,16 @@ func LayersFromOptions(ctx context.Context, o *service.TraceOptions) []string {
 	for _, x := range o.PerfettoConfig.GetDataSources() {
 		if layer, err := layout.LayerFromDataSource(x.GetConfig().GetName()); err == nil {
 			if _, err := layout.LibraryFromLayerName(layer); err == nil {
-				ret = append(ret, layer)
+				skip := false
```

Comment:
I think this is correct, but can I recommend
```
Outer:
  for _, x := range o.PerfettoConfig.GetDataSources() {
    if layer, err := layout.LayerFromDataSource(x.GetConfig().GetName()); err == nil {
	 if _, err := layout.LibraryFromLayerName(layer); err == nil {
            for _, l := range ret {
	        if l == layer {
		   continue Outer
		}
             }
             ret = append(ret, layer)
          }
    }
  }
```

Reply:
Done.  I did not know that was an option!